libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -533,6 +538,14 @@ class YulUtilFunctions
 	/// signature: (array, newLen)
 	std::string resizeDynamicByteArrayFunction(ArrayType const& _type);
 
+	/// @returns the name of a function that cleans up elements of a storage byte array starting from pos
+	/// it will not copy elements in case of transformation to short byte array, and will not change array length
+	/// In case of pos is greater than len, doesn't do anything
+	/// In case of short byte array (< 32 bytes) doesn't do anything
+	/// Cleans up from first unused slot
```

Comment:
```suggestion
	/// @returns the name of a function that cleans up elements of a storage byte array starting from pos.
	/// It will not copy elements in case of transformation to short byte array, and will not change array length.
	/// In case of pos is greater than len, doesn't do anything.
	/// In case of short byte array (< 32 bytes) doesn't do anything.
	/// Cleans up from first unused slot.
```